renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": [
+    "config:base",
+    ":gitSignOff",
+    ":dependencyDashboard",
+    ":prHourlyLimitNone",
+    ":rebaseStalePrs",
+    ":maintainLockFilesMonthly"
+  ],
+  "includeForks": true
+}
```

Comment:
I wonder if we could just use the GitHub-native Dependabot, which seems to [support Dockerfiles](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/about-dependabot-version-updates#supported-repositories-and-ecosystems) (opened #6 for that).

Reply:
In my experience, dependabot is less configurable than renovate. I just stole the config we use over at https://github.com/hedgedoc